### DISCUSSION TITLE
Volume for voice navigation does not adapt to media volume

### DISCUSSION THREAD

**Discussion by RHoffi** [devemux86/cruiser#152] [Q&A] [answered]
Set the voice navigation to use the music volume.
Set the volume  for navigation to 60%
At first everything works as expected.
After a Stop i stopp the navigation .
Then started again and music is adjusted directly from my bluetooth headset to a lower volume, the navigation volume remains and is much louder than the music. Problem can only be solved with restart of Cruiser.
Any Ideas or help?

**Comment by devemux86** [comment, marked as answer]
@RHoffi thanks for the detailed report.

> Set the voice navigation to use the music volume.

You mean the "Settings | Voice guidance | Voice guidance output | Media / music audio"?

> Set the volume for navigation to 60%

You mean the "Settings | Voice guidance | Voice volume"?

> After a Stop i stop the navigation.

Pause (tap) or Stop (long press) the navigation button?

When we set a fixed value, e.g. 60% in "Settings | Voice guidance | Voice volume",
the voice volume does not follow the system volume, it is fixed at 60% of maximum.

Have you tried to set it back to the default "System" value?
(drag the volume slider to the left)

<img src="https://user-images.githubusercontent.com/3484020/236543700-6db854fc-2d05-4bcf-a8cf-c887e7965435.jpg" width="33%"/>

**Reply by RHoffi** [reply]
Your assumptions are correct. Tomorrow I will navigate back to home. Try to set the volume according to your Suggestion. Everything Else works fine without any issues. Great work.

**Reply by devemux86** [reply]
Note that it is the _60% of the maximum volume_ (of the selected output).
(not the 60% of the current system volume)

**Reply by devemux86** [reply]
@RHoffi please let me know how you will hear it!

**Reply by RHoffi** [reply]
With setting it to System its perfect to me. Didnt know that the left of volume is system. I thought it was 0%.

**Reply by devemux86** [reply]
@RHoffi thanks for the feedback!

> Didnt know that the left of volume is system. I thought it was 0%.

Some settings have “Off” or “System” at the far left,
to avoid adding extra settings for the on / off switch.

We can probably improve the interface for such settings. :slightly_smiling_face:
